requirements.txt

```diff
@@ -1,6 +1,5 @@
 doc8==0.8.1
 docutils==0.16.0
-pip==22.0.2
```

Comment:
I unpinned pip because in CI this is coming from `setup-python` in the first place. This is misleadingly placed imo